src/containers/SubmitSteps/Questionary/Step1c/Step1c.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import usePortal from 'react-useportal';
+import { useTranslation } from 'react-i18next';
+
+// Form
+import { useForm, Controller } from 'react-hook-form';
+import { useStateMachine } from 'little-state-machine';
+import { yupResolver } from '@hookform/resolvers';
+import { ErrorMessage } from '@hookform/error-message';
+import * as Yup from 'yup';
+
+// Helper
+import { getPatientId, getCountry } from 'helper/stepsDefinitions';
+
+// Update Action
+import { updateAction } from 'utils/wizard';
+
+// Header Control
+import useHeaderContext from 'hooks/useHeaderContext';
+
+// Utils
+import { scrollToTop } from 'helper/scrollHelper';
+
+// Components
+import OptionList from 'components/OptionList';
+import DatePicker from 'components/DatePicker';
+import WizardButtons from 'components/WizardButtons';
+import ProgressIndicator from 'components/ProgressIndicator';
+
+// Styles
+import {
+  QuestionText, MainContainer, QuestionAllApply,
+} from '../style';
+
+const schemaWithoutPatient = Yup.object({
+  antigenTestDate: Yup.date().required(),
+  antigenTestResult: Yup.string().required(),
+}).defined();
+
+type Step1cType = Yup.InferType<typeof schemaWithoutPatient>;
+
+const Step1c = ({
+  previousStep,
+  nextStep,
+  storeKey,
+  metadata,
+}: Wizard.StepProps) => {
+  // Hooks
+  const { Portal } = usePortal({
+    bindTo: document && document.getElementById('wizard-buttons') as HTMLDivElement,
+  });
+  const {
+    setDoGoBack, setTitle, setSubtitle, setType,
+  } = useHeaderContext();
+  const history = useHistory();
+  const { t, i18n } = useTranslation();
+  const { state, action } = useStateMachine(updateAction(storeKey));
+  const patientId = getPatientId();
+  const country = getCountry();
+
+  // States
+  const [activeStep, setActiveStep] = React.useState(true);
+
+  // Form
+  const {
+    control, handleSubmit, formState,
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: state?.[storeKey],
+    context: {
+      country,
+    },
+    resolver: yupResolver(schemaWithoutPatient),
+  });
+  const { errors, isValid } = formState;
+
+  // Callbacks
+  const handleDoBack = React.useCallback(() => {
+    setActiveStep(false);
+    if (previousStep) {
+      history.push(previousStep);
+    } else {
+      history.goBack();
+    }
+  }, [history, previousStep]);
+
+  // Effects
+  useEffect(() => {
+    scrollToTop();
+    if (patientId) {
+      setTitle('');
+    } else {
+      setTitle(t('questionary:testTaken.title'));
+    }
+    if (patientId) {
+      setType('tertiary');
+    } else {
+      setType('primary');
+    }
```

Comment:
You can join both on one if
```suggestion
    if (patientId) {
      setTitle('');
      setType('tertiary');
    } else {
      setTitle(t('questionary:testTaken.title'));
      setType('primary');
    }
```